### PR TITLE
Hosted payments

### PR DESCRIPTION
**Pull request by lcavadas**

### CODE REVIEW

File: wp-e-commerce/wpsc-merchants/simplify.commerce.merchant.php [hunk 1]
```diff
@@ -211,152 +212,123 @@ function form_simplify() {
 	</tr>
 	<tr>
 		<td>
-			<label for="simplify_sandbox_private_key">' . __( 'Sandbox - Private Key:', 'wpsc' ) . '</label>
+			<label for="simplify_sandbox_private_key">' . __('Sandbox - Private Key:', 'wpsc') . '</label>
 		</td>
 		<td>
-			<input type="password" name="Simplify[sandbox_private_key]" id="simplify_sandbox_private_key" value="' . get_option( "simplify_sandbox_private_key" ) . '" size="30" size="30" style="width:500px;"/>
+			<input type="password" name="Simplify[sandbox_private_key]" id="simplify_sandbox_private_key" value="' . get_option("simplify_sandbox_private_key") . '" size="30" size="30" style="width:500px;"/>
```
Comment (siddii) [inline wp-e-commerce/wpsc-merchants/simplify.commerce.merchant.php]: The key lengths can go over 50.

Comment (siddii) [inline wp-e-commerce/wpsc-merchants/simplify.commerce.merchant.php]: I was checking my keys. One is 64 & the other one is 53

File: wp-e-commerce/wpsc-merchants/simplify.commerce.merchant.php [hunk 2]
```diff
@@ -34,169 +34,170 @@
  */
 
 // Simplify Commerce PHP SDK 
-require_once(WPSC_FILE_PATH.'/wpsc-merchants/library/simplifycommerce/Simplify.php');
+require_once(WPSC_FILE_PATH . '/wpsc-merchants/library/simplifycommerce/Simplify.php');
 
 // Using WPSC API v2.0
 $nzshpcrt_gateways[$num]['api_version'] = 2.0;
-$nzshpcrt_gateways[$num]['name'] = __( 'Simplify Commerce v1.0.1', 'wpsc' );
+$nzshpcrt_gateways[$num]['name'] = __('Simplify Commerce v1.0.1', 'wpsc');
```
Comment (siddii) [inline wp-e-commerce/wpsc-merchants/simplify.commerce.merchant.php]: please use v1.1.0

File: wp-e-commerce/wpsc-merchants/simplify.commerce.merchant.php (hunk 1 above)
Comment (lcavadas) [inline wp-e-commerce/wpsc-merchants/simplify.commerce.merchant.php]: The size property has nothing to do with the allowed length of the content. It's about the width of the element.

Comment (tychobrailleur) [inline wp-e-commerce/wpsc-merchants/simplify.commerce.merchant.php]: @lcavadas Does it need to be repeated twice, though? ;-)

Comment (lcavadas) [inline wp-e-commerce/wpsc-merchants/simplify.commerce.merchant.php]: I'd say it doesn't, but I didn't change this... I just reformatted it, it's a whitespace change.

That said I'll remove the duplication.

**Pull request merged**